python-plugin/src/test/resources/checks/AvoidListComprehensionInIterations.py

```diff
@@ -0,0 +1,39 @@
+def non_compliant_example_basic():
+    for var in [var2 for var2 in range(1000)]: # Noncompliant {{Use generator comprehension instead of list comprehension in for loop declaration}}
+        print(var)
+
+def non_compliant_example_enumerate():
+    for idx, var in enumerate([var2 for var2 in range(1000)]): # Noncompliant {{Use generator comprehension instead of list comprehension in for loop declaration}}
+        print(var)
+
+def non_compliant_example_zip():
+    for var, var_ in zip([var2 for var2 in range(1000)], [var2 for var2 in range(1000)]): # Noncompliant {{Use generator comprehension instead of list comprehension in for loop declaration}} {{Use generator comprehension instead of list comprehension in for loop declaration}}
+        print(var)
+
+def non_compliant_example_zip():
```

Comment:
there is two same methods named `non_compliant_example_zip`. why ?

Reply:
Duplicated function removed. Same for `ecoCode-python-test-project`.